packages/fx-core/src/component/driver/aad/utility/collaboration.ts

```diff
@@ -0,0 +1,102 @@
+import { Context, FxError, M365TokenProvider, Result, err, ok } from "@microsoft/teamsfx-api";
+import { Service } from "typedi";
+import { hooks } from "@feathersjs/hooks/lib";
+import { AadOwner, ResourcePermission } from "../../../../common/permissionInterface";
+import { AadAppClient } from "./aadAppClient";
+import { permissionsKeys } from "./constants";
+import { addStartAndEndTelemetry } from "../../middleware/addStartAndEndTelemetry";
+import axios from "axios";
+import { HttpClientError, HttpServerError, UnhandledError } from "../../../../error/common";
+
+const EventName = {
+  grantPermission: "grant-permission",
+  listCollaborator: "list-collaborator",
+  checkPermission: "check-permission",
+};
+const componentName = "fx-resource-aad-app-for-teams";
```

Comment:
minor, no need to fix in this PR: when we refactor telemetry, we can see whether we want to retire the legacy component names, though it brings problems to BI report